examples/angular/src/pages/ui/components/authenticator/sign-in-sms-mfa/sign-in-sms-mfa.component.html

```diff
@@ -4,7 +4,7 @@
     let-user="user"
     let-signOut="signOut"
   >
-    <h2>Welcome, {{ user.attributes?.name }}!</h2>
+    <h2>Welcome, {{ user.attributes?.email }}!</h2>
```

Comment:
Does one of the available attributes map to what we are currently using for `name`? Maybe I'm mistaken, but reading this as a breaking change

Reply:
I believe in this case the app was broken in prod as well